includes/functions.php

```diff
@@ -2168,6 +2168,7 @@ function tsml_to_css_classes($types, $prefix = 'type-')
     }
 
     $types = array_map('strtolower', $types);
+    $types = array_map('trim', $types);
```

Comment:
@kiyote33 Hey Wayne I know no one asked, but locally I saw this was spitting out odd classes like:
`type-o type- onl type- en type-onl`
because types often seem to have whitespace around them. Hope this is OK.

Reply:
No I hadn't seen that. Thx for fixing...